pkg/server/controllers.go

```diff
@@ -1274,7 +1274,7 @@ func (s *Server) installSyncTargetController(ctx context.Context, config *rest.C
 	c := synctargetcontroller.NewController(
 		kcpClusterClient,
 		s.KcpSharedInformerFactory.Workload().V1alpha1().SyncTargets(),
-		s.KcpSharedInformerFactory.Core().V1alpha1().Shards(),
+		s.CacheKcpSharedInformerFactory.Core().V1alpha1().Shards(),
```

Comment:
still seeing some tests failing in `test/e2e/syncer pkg`. I will revert the change to `synctargetcontroller` controller for now.